doc/source/configuration/cephadm.rst

```diff
@@ -107,7 +107,7 @@ OSD specification
 ~~~~~~~~~~~~~~~~~
 
 The following example is a basic OSD spec that adds OSDs for all
-available disks:
+available disks with encryption at rest:
```

Comment:
Is this definitely something we want to suggest as default? Mainly wondering if there's any performance impact.

Reply:
I think it's fairly standard in our deployments these days. No doubt there will be some performance impact, but unsure how much.